pages/docs/concepts/asyncapi-document/reply-info.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Adding reply Info
+weight: 210
+---
+
+Request/Reply is a common messaging or communication pattern where one component or service, known as the requestor, sends a request to another component or service, known as the replier and waits for a response. This pattern is often used when a component needs to initiate an action and receive a specific response in return either synchronously or asynchronously. This communication pattern is typically asynchronous, meaning that the requester does not block and wait for an immediate response. Instead, it can continue processing other tasks or even send out other requests while waiting for the reply to arrive. 
+
+In case of multiple requests made, each request is processed independently and sends the reply/response to the corresponding requestor when ready.
+
+The request/reply pattern in AsyncAPI works in the same fashion while supporting all the different sub-patterns. Irrespective of the sub-pattern you would like to implement, the Request/reply pattern can be implemented in the asyncAPI document using actions in the `Operation` object.
```

Comment:
```suggestion
The request/reply pattern in AsyncAPI works in the same fashion while supporting all the different sub-patterns. Irrespective of the sub-pattern you would like to implement, the Request/reply pattern can be implemented in the AsyncAPI document using actions in the `Operation` object.
```

> can be implemented in the AsyncAPI document using actions

what do you mean like? 

- it is not like "implemented" but rather "represented"
- and what actions have to do here, it is more a reply?